clang-tools-extra/clang-doc/Mapper.cpp

```diff
@@ -53,30 +80,34 @@ template <typename T> bool MapASTVisitor::mapDecl(const T *D) {
 }
 
 bool MapASTVisitor::VisitNamespaceDecl(const NamespaceDecl *D) {
-  return mapDecl(D);
+  return mapDecl(D, true);
```

Comment:
for constant params you should have a comment, per our style guide.
```suggestion
  return mapDecl(D, /*isDefinition=*/true);
```